compio-fs/tests/file.rs

```diff
@@ -75,19 +74,23 @@ async fn cancel_read() {
     read_hello(&file).await;
 }
 
+#[cfg(unix)]
 #[compio_macros::test]
 async fn timeout_read() {
-    let mut tempfile = tempfile();
-    tempfile.write_all(HELLO).unwrap();
+    use std::time::Duration;
 
-    let file = File::open(tempfile.path()).await.unwrap();
+    use compio_fs::pipe::anonymous;
+    use compio_io::AsyncReadExt;
+    use compio_runtime::time::timeout;
 
-    // Read a file with timeout.
-    let _ = timeout(Duration::from_nanos(1), async { read_hello(&file).await })
-        .await
-        .unwrap_err();
+    let (mut rx, _) = anonymous().unwrap();
 
-    read_hello(&file).await;
```

Comment:
This one is originally used to test read operation after the former one is cancelled.

Reply:
But we already have a canceled test?

Reply:
Well, fair.